incubator/hnc/e2e/tests/e2e_test.go

```diff
@@ -19,3 +19,15 @@ func TestE2e(t *testing.T) {
 		"HNC Suite",
 		[]Reporter{printer.NewlineReporter{}})
 }
+
+func  runSuccess(cmdln ...string) {
+	var args []string
+	for _, subcmdln := range cmdln {
+		args = append(args, strings.Split(subcmdln, " ")...)
+	}
+	GinkgoT().Log(args)
+	cmd := exec.Command(args[0], args[1:]...)
+	stdout, err := cmd.CombinedOutput()
+	GinkgoT().Log(string(stdout))
```

Comment:
Consider differentiating somehow between the command (line 28) and the output (here).

Reply:
added prefix messages